content/authors/rachel-delange/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+# View this page at https://digital.gov/authors/rachel-delange
+# Learn how to edit our pages at https://workflow.digital.gov
+
+# slug — the specific user-id for an author.
+slug: rachel-delange
+display_name: "Rachel DeLange"
+first_name: "Rachel"
+last_name: "DeLange"
+
+# List your pronoun(s) if you want them displayed alongside your name. If blank, we'll use just your name. Learn more http://mypronouns.org
+pronoun: ""
+
+# Email — If you include an email address, it will be displayed on your profile page
+email: 
+
+# Bio — keep it under 50 words
+bio: ""
+
+# bio_url — Where can people learn more about your work? Provide a full URL [e.g. 'https://www.example.gov/']
+bio_url: 
+
+# Agency Full Name [e.g. U.S. General Services Administration]
+agency_full_name: "Homeland Security and Emergency Management Agency"
```

Comment:
This isn't a federal agency. It looks like she works for the Washington, D.C. Homeland Security and Emergency Management Agency
https://hsema.dc.gov/

Reply:
gotcha - i will edit that - thanks for the head's up and sorry for the confusion there